src/App/App.jsx

```
@@ -36,9 +36,10 @@
 import { routes } from './routes'
 
 function App({ dexieCurrentUserInstance }) {
+  console.log('I want to see an error')
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)